src/maintainer/knowledge_base.rst

```diff
@@ -1518,3 +1518,27 @@ The tl;dr here is that conda sorts as follows:
 So make sure that you **tag** your package in such a way that the package name
 that conda-build spits out will sort the package uploaded with an ``rc`` label
 higher than the package uploaded with the ``dev`` label.
+
+Migrators and Migrations
+========================
+
+When any changes are made in the global pinnings of a package, then the entire stack of the packages that depend on that package would need to be updated and rebuilt.
```

Comment:
Not all packages that depend on a pinned package need to be rebuilt. We only need to rebuilt those that impose an ABI dependency (usually via linking / dynamic libraries, but sometimes also header libraries like pybind11). I am not sure of what the right terminology is here, though, but the rebuild only concerns packages which depend on pinned packages in their `host` section.

```suggestion
When any changes are made in the global pinnings of a package, then the entire stack of the packages that need that package on their `host` section would need to be updated and rebuilt.
```